configuredChains.yaml

```diff
@@ -558,6 +558,20 @@ lumnetwork:
       uid-gid: 1025:1025
   no-host-mount: false
 
+namada:
+  name: namada
+  type: namada
+  bin: namada
+  bech32-prefix: tnam
+  denom: nam
+  gas-prices: 0.000001tnam1qxgfw7myv4dh0qna4hq0xdg6lx77fzl7dcem8h7e
```

Comment:
just want to confirm that gas-prices use this tnam1 (looks like a bech32 address) instead of the `nam` denom directly

Reply:
`nam` is just an alias. The address is required.